config.default.yaml

```diff
@@ -270,10 +270,23 @@ industry:
   MWh_elec_per_tNH3_electrolysis: 1.17 # from https://doi.org/10.1016/j.joule.2018.04.017 Table 13 (air separation and HB)
   NH3_process_emissions: 24.5 # in MtCO2/a from SMR for H2 production for NH3 from UNFCCC for 2015 for EU28
   petrochemical_process_emissions: 25.5 # in MtCO2/a for petrochemical and other from UNFCCC for 2015 for EU28
-  HVC_primary_fraction: 1.0 #fraction of current non-ammonia basic chemicals produced via primary route
+  HVC_primary_fraction: 1. # fraction of today's HVC produced via primary route
+  HVC_mechanical_recycling_fraction: 0. # fraction of today's HVC produced via mechanical recycling
+  HVC_chemical_recycling_fraction: 0. # fraction of today's HVC produced via chemical recycling
```

Comment:
Should we set defaults for recycling where not all plastics are primary?

E.g. mechanical (20-30%), chemical (15%), reuse (5-10%) recycling.

Long-term it should be possible to set an exogenous pathway.

In the current implementation, would reuse (no energy demand) be the 1 - primary - mechanical - chemical?

Reply:
Yes, that's a good idea, but probably it should be a path 2020-50 with increasing rates of recycling and reuse.